File: src/test/java/kitchenpos/products/tobe/infra/FakeProfanities.java
package kitchenpos.products.tobe.infra;

import kitchenpos.products.tobe.domain.Profanities;

import java.util.Arrays;
import java.util.List;

public class FakeProfanities implements Profanities {

    private static final List<String> profanities;

    static {
        profanities = Arrays.asList("욕설", "비속어");
    }

    @Override
    public boolean contains(final String text) {
        return profanities.stream()
                .anyMatch(profanity -> text.contains(profanity));
    }
}

File: src/test/kotlin/kitchenpos/product/tobe/domain/FakeProfanities.kt
package kitchenpos.product.tobe.domain

class FakeProfanities(
    private val profanities: List<String> = emptyList()
) : Profanities {

    override fun contains(profanity: String): Boolean {
        return profanities.any { profanity.contains(it) }
    }
}
